client/blocks/feedback/edit.js

```diff
@@ -470,7 +504,24 @@ const EditFeedbackBlock = ( props ) => {
 												'Collect your own feedback with Crowdsignal',
 												'crowdsignal-forms'
 											) }
-										/>
+										>
+											<Tooltip
+												text={ promoteLink }
+												position="top center"
+											>
+												<a
+													href="https://crowdsignal.com/pricing"
+													target="_blank"
+													rel="noopener noreferrer"
+													className="crowdsignal-forms__branding-promote"
+												>
+													{ __(
+														'Hide',
+														'crowdsignal-forms'
+													) }
+												</a>
+											</Tooltip>
```

Comment:
You may have considered this already, but one possibility would be to extract this to a separate module, to be used for both `feedback/edit.js` and `nps/edit.js`. That would still keep it out of the frontend, and should be handled just fine by the build process.

Reply:
Yeah, I think I was mostly dashing and slashing until I got it out, likely suspect to get into a module on its own.

Reply:
I moved the tooltip to a separate component so to reuse it. Thanks for the suggestion!